src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_fr.properties

```
@@ -193,14 +193,14 @@ codeCompletionDescription= Quand utiliser l''auto-complétion pour le code R dan
 codeCompletionEnum_always=Automatiquement
 codeCompletionEnum_never=Jamais
 codeCompletionEnum_triggered=Lorsque déclenché ($, : :)
-codeCompletionEnum_manual=Manuellement (touche de tabulation)
+codeCompletionEnum_manual=Manuellement (Tab, Ctrl + Space)
```

nit to match the lack of spaces in the `codeCompletionOtherEnum_manual` string before and after the `+`.
```suggestion
codeCompletionEnum_manual=Manuellement (Tab, Ctrl+Space)
```
(see the corresponding English string as well)